### PR TITLE
Updates to crescendo merge

### PR DESCRIPTION
**Pull request by sigalrmp**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -78,6 +78,15 @@ dependencies {
 
     // Monologue
     implementation 'com.github.shueja:Monologue:+'
+
+    // Gson
+    // Remove this once choreolib changes get upstreamed
+    implementation 'com.google.code.gson:gson:2.10.1'
```
Comment (sigalrmp) [inline build.gradle:84]: not sure if this is necessary

File: .DataLogTool/datalogtool.json [hunk 2]
```diff
@@ -1,9 +0,0 @@
-{
```
Comment (sigalrmp) [inline .DataLogTool/datalogtool.json:1]: not sure if straight up deleting this is right, but it a path specific to Ivan's laptop definitely shouldn't be in Hydrogen

File: build.gradle (hunk 1 above)
Comment (Yxhej) [reply build.gradle:84]: Pretty sure Choreo requires it for all (de)serialization and json file creation, which we will probably want to keep for the time being.

**Review by Yxhej — APPROVED** [review]

File: .DataLogTool/datalogtool.json (hunk 2 above)
Comment (Yxhej) [reply .DataLogTool/datalogtool.json:1]: It'll be regenerated, so just leave it blank

**Pull request merged**